downgrade

```diff
@@ -179,7 +191,8 @@ search_packages() {
     fi
 
     # shellcheck disable=SC2086
-    find -L "${PACMAN_CACHE[@]}" -maxdepth "$DOWNGRADE_MAXDEPTH" -regextype posix-extended -regex ".*/$pkgfile_re"
+    find -L "${PACMAN_CACHE[@]}" -maxdepth "$DOWNGRADE_MAXDEPTH" \
+      -regextype posix-extended -regex ".*/$pkgfile_re" 2>/dev/null
```

Comment:
What prompted the `2>/dev/null`?

Reply:
I wrongly typed a path in `PACMAN_CACHE` and that resulted in `find` throwing an error message. I thought it would be good to route this error message away, as we already did in other cases within `downgrade`. WDYT?

Reply:
I try not to avoid suppressing errors unless there's a reason. If an end-user made a mistake in `--pacman-cache`, would it be better to show them what `find` said about it or hide it? I don't know the answer, but that's the question I'd ask.

> as we already did in other cases

This is definitely a case-by-case decision IMO, but if you describe the other cases, maybe there is a generally applicable way to decide for this one.

Reply:
Yes true. The other case where I had used `2>/dev/null` was in the `grep` call for reading `downgrade.conf`.  But there it makes sense, because a user could opt not to have a `downgrade.conf` file and should not receive constant error messages about it being missing; since it is ultimately optional.

But I guess the argument is different for `find`, in that an error message could be useful there. In any case, I dropped this commit pertaining to `find` and left it as it was before.